docs/en/compute/Presto.md

```diff
@@ -20,6 +20,22 @@ latency especially when data is remote or network is slow or congested.
 * Table of Contents
 {:toc}
 
+## Using Presto with the Alluxio Catalog Service
+Currently, there are 2 ways to enable Presto to interact with Alluxio:
+* Presto interacts with the [Alluxio Catalog Service]({{ '/en/core-services/Catalog.html' | relativize_url }})
+* Presto interacts with the Hive Metastore (updated with Alluxio locations)
```

Comment:
I think that "updated with Alluxio locations" is a little vague.

Suggestion "with table definitions updated to use Alluxio paths"?

Reply:
done